cmd/jaeger/ingester-remote-storage.yaml

```diff
@@ -12,6 +12,11 @@ service:
       level: debug
 
 extensions:
+  healthcheckv2:
+    use_v2: true
+    http:
+      endpoint: "0.0.0.0:13133"
```

Comment:
will you not get a port conflict if both collector and ingester use the same port for HC extension? E.g. we bumped the metrics port from 8888 to 8889 above.